redfish-core/lib/log_services.hpp

```diff
@@ -699,6 +741,19 @@ inline void
                                 size = *sizePtr;
                                 break;
                             }
+                            else if (propertyMap.first == "InitiatedBy")
+                            {
+                                const std::string* session =
+                                    std::get_if<std::string>(
+                                        &propertyMap.second);
+                                if (session == nullptr)
+                                {
+                                    messages::internalError(asyncResp->res);
+                                    break;
```

Comment:
Typically this is a return, I see you are following what else is in this file so I guess it is okay

Reply:
yeah, will take this in a new commit may be..